### ISSUE TITLE
express-validator passing assert function into req object for testing

### ISSUE THREAD

**Issue by kuroni** [ctavan/express-validator#29]
so, my issue is this. I am using the the validator stuff like "req.assert("gameName", "You must supply a game name").notEmpty();" in one of my controllers. in my test file, i send in req object of {assert: function(x){}, ....} so that in my controller, it can call the assert method. however i keep getting: TypeError: Object  has no method 'notEmpty'. so i tried replacing the function with assert(). but then i get: undefined is not equal to true.

so basically my problem is for my testing, how do i pass in the assert method so it works with my controller.

fyi, i am using node.js as my framework, and mocha for testing

**Comment by rustybailey** [comment]
Another issue about testing (https://github.com/ctavan/express-validator/issues/123) mentions this post: http://jaketrent.com/post/testing-express-validator/
You could also try sinon.js for stubbing: http://sinonjs.org/
